config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml

```diff
@@ -0,0 +1,39 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-mkdocs-job
+      decorate: true
+      run_if_changed: 'images/mkdocs'
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.2
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                export PATH=$PATH:/usr/local/bin
+                export KUBECONFIG=/etc/kubeconfig/config
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                pushd images/mkdocs
+                export VERSION=( `cat version.txt` )
+                declare -a VERSION
+                echo ${VERSION[@]}
```

Comment:
before this following code needed, https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml#L36:L59 to generate a proper kubeconfig file to build the image